lisa/tools/lsblk.py

```diff
@@ -290,6 +305,21 @@ def find_disk_by_mountpoint(
 
         raise LisaException(f"Could not find disk with mountpoint {mountpoint}")
 
+    def find_partition_by_mountpoint(
+        self, mountpoint: str, force_run: bool = False
+    ) -> PartitionInfo:
+        disk = self.find_disk_by_mountpoint(mountpoint, force_run=force_run)
+
+        if disk is None:
```

Comment:
It doesn't need, because find_disk_by_mountpoint raises exception already.

Reply:
My bad. removed it.